STACK-UP-3/champs-backend
# #171648194 Create accommodation facility

#### Description
This feature creates accommodation on Barefoot Nomad So that, travelers can see the accommodation facility and its room availability.

#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test 
Setup:

 1. Clone this repo on your machine
 2. Add `.env` file by referring to `.env.example`
 3. Go in the `swagger` file and change `host` address to `localhost:3000`
 4. Run `npm i ` to install dependencies
 5. Do `npm run serve` command

Test:

 1. Open your preferred testing tool, such postman 
 2. Enter this link in the URL `localhost:3000/api/v1/signin` to sign in,
 3. Sign in by providing your email and password and copy the given token,
 4. Provide token in the headers, like `token` as key and paste your token as the value,
 5. Enter this link in the URL `localhost:3000/api/v1/accommodation` for registering accommodation,
 6. Specify the request method as `post`,
 7. Enter all required information to register an accommodation in the body if your request,
 8. Then hit the send button.


Give instructions on how reviewers can verify your work
- You will get the response if you have registered successfully or what's wrong

Screenshots or GIFs
**Request**:
![image](https://user-images.githubusercontent.com/11549858/80805789-373a4e00-8bb9-11ea-82f3-2b3553bd5338.png)

**Response**
![image](https://user-images.githubusercontent.com/11549858/80806018-cf383780-8bb9-11ea-9069-c36916d5f43a.png)

#### STORY ID
[Finishes #171648194]
